src/index.json

```diff
@@ -328,6 +328,51 @@
                     "summary": "An Azure CLI Extension to manage appservice resources",
                     "version": "0.2.8"
                 }
+            },
+            {
+                "filename": "webapp-0.2.9-py2.py3-none-any.whl",
+                "sha256Digest": "f75b3e09cd3d99a0aaf47d9cf1f3792f6016bbbab4815d42afe6c55e5e306e1f",
+                "downloadUrl": "https://github.com/panchagnula/azure-cli-extensions/raw/sisirap-extensions-whl/dist/webapp-0.2.9-py2.py3-none-any.whl",
+                "metadata": {
+                    "azext.isPreview": true,
+                    "azext.minCliCoreVersion": "2.0.46",
+                    "classifiers": [
+                        "Development Status :: 4 - Beta",
+                        "Intended Audience :: Developers",
+                        "Intended Audience :: System Administrators",
+                        "Programming Language :: Python",
+                        "Programming Language :: Python :: 2",
+                        "Programming Language :: Python :: 2.7",
+                        "Programming Language :: Python :: 3",
+                        "Programming Language :: Python :: 3.4",
+                        "Programming Language :: Python :: 3.5",
+                        "Programming Language :: Python :: 3.6",
+                        "License :: OSI Approved :: MIT License"
+                    ],
+                    "extensions": {
+                        "python.details": {
+                            "contacts": [
+                                {
+                                    "email": "sisirap@microsoft.com",
+                                    "name": "Sisira Panchagnula, Lukasz Stempniewicz",
```

Comment:
Is Lukasz also a code owner? If so, we should add him to the [codeowners.txt ](https://github.com/Azure/azure-cli-extensions/blob/master/.github/CODEOWNERS)

Reply:
Lukasz doesn't do this work anymore , should i just remove him?

Reply:
Yes, this would be good